llvm/docs/LangRef.rst

```diff
@@ -19135,8 +19137,10 @@ If the result value cannot be precisely represented in the given scale, the
 value is rounded up or down to the closest representable value. The rounding
 direction is unspecified.
 
-It is undefined behavior if the result value does not fit within the range of
-the fixed point type, or if the second argument is zero.
+If the result value does not fit within the range of the fixed point type, the
+result is poison.
```

Comment:
Can udiv.fix even produce a value that does not fit the range?

Reply:
Can it overflow if you divide by a fractional value? e.g. i8 udiv.fix(i8 255, i8 1, i8 1) = 127.5 / 0.5 = 255, which > 127.5?

Reply:
Ah yeah, you're right.